packages/office-ui-fabric-react/src/components/GroupedList/GroupHeader.tsx

```diff
@@ -94,6 +99,7 @@ export class GroupHeader extends BaseComponent<IGroupDividerProps, IGroupHeaderS
             type="button"
             className={css('ms-GroupHeader-expand', styles.expand)}
             onClick={this._onToggleCollapse}
+            {...expandCollapseButtonProps}
```

Comment:
small nit, but might as well match the terminology here. expandButtonProps instead of expandCollapseButtonProps